{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/urls.py

```diff
@@ -51,19 +51,24 @@
     from django.views import defaults as dj_default_views
```

Comment:
These changes look good. What are your thoughts on separating this out and put it in the base module to include? Makes the urls file less long when there are new urls added to it.

Reply:
Sounds good. Will try to get it cleaned up a little, but not in this PR.

Reply:
👍 !